src/strings.rs

```diff
@@ -1,3 +1,5 @@
+pub const DEFAULT_PREFIX: &str = "!";
```

Comment:
I would say that we should change this to what is set in `.env.example` aka `-`.